crates/rethnet_evm_napi/test/evm/StateManager.ts

```diff
@@ -1,7 +1,7 @@
 import { expect } from "chai";
 import { Address, KECCAK256_NULL } from "@nomicfoundation/ethereumjs-util";
 
-import { Account, Bytecode, RethnetContext, State } from "../..";
+import { Account, Bytecode, RethnetContext, State } from "../../index.js";
```

Comment:
Why does it no longer recognise the `*.d.ts`?

Reply:
Not sure tbh but it was failing in the CI without that change so I would rather not touch it 😅